lang/sql/sql.talon

```diff
@@ -0,0 +1,38 @@
+mode: command
+and mode: user.sql
+mode: command
+and mode: user.auto_lang
+and code.language: sql
+-
+tag(): user.code_generic
+tag(): user.code_operators
+
+# Some commands have spaces around them to avoid having to continually say 'space'.
+# The ones that don't have an initial space are expected to be at the start of a new line.
+# This could probably be done in a better way.
+select: "SELECT "
+star: "* "
```

Comment:
Note that `star` is already defined as a command as a symbol key. This would be overriding that.

Reply:
Agree shadowing commands for basic keys is not a good idea. One option might be to define a single command for `SELECT * FROM` the way I do:

https://github.com/nriley/knausj_talon/blob/nriley/lang/sql/sql.talon#L12